src/PasswordResetForm/PasswordResetForm.test.js

```diff
@@ -42,4 +42,14 @@ describe("PasswordResetForm", () => {
       ).toBeInTheDocument()
     );
   });
+  it("displays error message to user on validation fail", async () => {
+    const elements = setup();
+    userEvent.type(elements.inputs.email, "joe.bloggs"); // missing password
```

Comment:
shouldn't userEvent be wrapped in act @mattcorner ?

Reply:
You'd think so, but not in this case because the form validation is only applied onSubmit, not onTouch like it is in something like RegistrationForm.

Reply:
Also, it's testing wrong email not missing password as the comment suggests. Don't think there is a password on this form.